types/k-bucket/index.d.ts

```diff
@@ -0,0 +1,153 @@
+// Type definitions for k-bucket 5.0
+// Project: https://github.com/tristanls/k-bucket#readme
+// Definitions by: tomonari-t <https://github.com/me>
```

Comment:
<!--
  1 failure:  This line should ...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/k-bucket/index.d.ts;
  Line: 3;
-->
- :no_entry_sign: This line should have your github username in it, not /me

Reply:
Right, thank you !